pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical and math expression (special symbols should be escaped):
+
+`expr {{ARG1}} {{operator}} {{ARG2}}`
+
+- Get position of the first character in 'string' that matches 'substring':
+
+`echo $(expr index {{string}} {{substring}})`
+
+- Extract part of the string:
+
+`echo $(expr substr {{string}} {{number_of_chr_position}} {{how_many_chrs_extract}}`
```

Comment:
It's a little unclear what "number_of_chr_position" means. Can't that parameter be named something simpler, clearer, or both?